<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home_window</class>
 <widget class="QMainWindow" name="home_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SML Maker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>17</number>
      </property>
      <item>
       <widget class="QPushButton" name="export_button">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>Export to SML</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="import_button">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>Import from SML</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="about_menu"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="export_to_sml_menu"/>
    <addaction name="import_from_sml_menu"/>
    <addaction name="separator"/>
    <addaction name="exit_menu"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="messageBar"/>
  <action name="about_menu">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="exit_menu">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="export_to_sml_menu">
   <property name="text">
    <string>Export to SML</string>
   </property>
  </action>
  <action name="import_from_sml_menu">
   <property name="text">
    <string>Import from SML</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
